crates/ruff_db/src/diagnostic/mod.rs

```diff
@@ -1390,6 +1401,7 @@ impl Default for DisplayDiagnosticConfig {
             show_fix_status: false,
             show_fix_diff: false,
             fix_applicability: Applicability::Safe,
+            program: Program::Ty,
```

Comment:
I think at this point, it might be useful to refactor this into a `new` method that takes in a `program: Program` as the sole parameter because there's no valid default value for this field and the caller should always provide the program for which this rendering is to be done for. What do you think?